apps/hosted-viewer-harness/src/app.tsx

```diff
@@ -0,0 +1,22 @@
+import { Heading, VStack } from '@chakra-ui/react';
+import React from 'react';
+
+import './styles/app.css';
+const { VITE_RTS_ACCOUNT_ID, VITE_RTS_STREAM_NAME } = import.meta.env;
+const App = () => {
+  return (
+    <VStack background="background" height="100%" padding="24px" spacing="24px" width="100%">
+      <Heading as="h1" fontSize="32px" test-id="harnessHeader">
+        Hosted Viewer Harness
+      </Heading>
+      <iframe
+        src={`http://localhost:5173?streamId=${VITE_RTS_ACCOUNT_ID}/${VITE_RTS_STREAM_NAME}`}
```

Comment:
nit: We _could_ do it this way, but if this is also hosted on netlify, then it makes sense to pull out something like the `APP_URL` from the `.env` file. During the build phase, we can inject the netlify URL.

Thoughts?

Reply:
oh actually, probably ok to leave it like this for now...

Reply:
@dubeyShivank Yeah, for right now this is meant for just development. We'll need to transform it at some point for deployment with QA and integration with CD. Do you guys tend to use the different .env files for different environments, such as .env.dev and .env.staging?